### DISCUSSION TITLE
Shell integration?

### DISCUSSION THREAD

**Discussion by dsully** [caio-ishikawa/slingshot#17] [Ideas]
Have you thought about shell integration? For example with fish + fzf: https://github.com/PatrickF1/fzf.fish

I can type `Ctrl+R` and can start searching my shell history.

**Comment by caio-ishikawa** [comment]
Interesting idea! I hadn't considered it but I'll look into it this week. Thank you.

**Comment by caio-ishikawa** [comment]
Hey, I added a simple integration with the fish shell today. You should be able to install with it `fisher install caio-ishikawa/slingshot`. The default binding to start it is Ctrl+S.

I appreciate the interest!
